web/tailwind.config.cjs

```diff
@@ -19,6 +19,14 @@ module.exports = {
 			fontFamily: {
 				'immich-title': ['Snowburst One', 'cursive']
 			}
+		},
+		screens: {
+			xs: '430px',
+			sm: '640px',
+			md: '768px',
+			lg: '1024px',
+			xl: '1280px',
+			'2xl': '1536px'
```

Comment:
You've defined an extra `xs` breakpoint besides the default ones. I think you're trying to target mobile devices specifically with such a small breakpoint, but I often find the `sm` breakpoint to be better suited. For screens in the 430-640px range everything already starts to look crammed together, so the `xs` breakpoint might not be needed at all.

Reply:
I think you're right. Changed in 41467184ee51e8bd04af67c649599384d2e465c4